api/modules/files/routes/read.js

```diff
@@ -71,21 +73,21 @@ module.exports = [{
     }
   }
 }, {
-  method: 'GET',
-  path: '/projects/{project_id}/files/tar',
+  method: `GET`,
+  path: `/projects/{project_id}/files/tar`,
   config: {
     pre: [
-      prereqs.confirmRecordExists(Model, {
-        mode: 'param',
-        requestKey: 'project_id',
-        columns: ['id', 'path']
+      Prerequisites.confirmRecordExists(FilesModel, {
+        mode: `param`,
+        requestKey: `project_id`,
+        columns: [`id`, `path`]
       }),
-      prereqs.validateUser(),
-      prereqs.validateOwnership()
+      Prerequisites.validateUser(),
+      Prerequisites.validateOwnership()
     ],
-    handler: controller.getAllAsTar.bind(controller),
-    description: 'Retrieve a tar file containing a collection of file objects that belong to a ' +
-    'single project object, based on `project_id`.',
+    handler: filesController.getAllAsTar.bind(filesController),
```

Comment:
`handler: args => filesController.getAllAsTar(...args),` although it feels odd that this can't just be a static function and needs explicit tying to a specific fileController instance. It suggests we might have unnecessary data capturing in the fileController class.